### ISSUE TITLE
Installation process on ubuntu 22.04

### ISSUE THREAD

**Issue by DavidJimenez10** [Piemontez/opencvflow#4]
Hi, I'm on Ubuntu 22.04
When I tried to install the project I get the following error

```bash
8186 timing command:install Completed in 139145ms
8187 verbose stack Error: command failed
8187 verbose stack     at ChildProcess.<anonymous> (/usr/share/nodejs/@npmcli/promise-spawn/index.js:64:27)
8187 verbose stack     at ChildProcess.emit (events.js:314:20)
8187 verbose stack     at maybeClose (internal/child_process.js:1022:16)
8187 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
8188 verbose pkgid opencvflow@0.9.3
8189 verbose cwd /home/djm/Documents/opencvflow
8190 verbose Linux 5.15.0-60-generic
8191 verbose argv "/usr/bin/node" "/usr/bin/npm" "install"
8192 verbose node v12.22.9
8193 verbose npm  v8.5.1
8194 error code 1
8195 error path /home/djm/Documents/opencvflow
8196 error command failed
8197 error command sh -c ts-node .erb/scripts/check-native-dep.js && electron-builder install-app-deps && cross-env NODE_ENV=development TS_NODE_TRANSPILE_ONLY=true webpack --config ./.erb/configs/webpack.config.renderer.dev.dll.ts && opencollective-postinstall
8198 verbose exit 1
8199 timing npm Completed in 139505ms
8200 verbose code 1
8201 error A complete log of this run can be found in:
8201 error     /home/djm/.npm/_logs/2023-02-11T19_42_16_782Z-debug-0.log
```
Any suggestions? Thanks in advance

**Comment by Piemontez** [comment]
I couldn't reproduce the error.

What version of the node you have installed?

**Comment by Piemontez** [comment]
The new available version (0.10.1) no longer uses Electron and its dependencies.
Many build errors were resolved with this adjustment.
